src/main.cpp

```diff
@@ -2809,6 +2809,7 @@ bool AcceptBlockHeader(const CBlockHeader& block, CValidationState& state, CBloc
     uint256 hash = block.GetHash();
     BlockMap::iterator miSelf = mapBlockIndex.find(hash);
     CBlockIndex *pindex = NULL;
+    if (hash != chainparams.GetConsensus().hashGenesisBlock) {
     if (miSelf != mapBlockIndex.end()) {
```

Comment:
yep, I want to avoid indenting the whole block to reduce the diff and potential for merge conflicts.
After all, at some point we will start running clang-format project wise right before forking for major releases. Is that still the plan @laanwj @sipa ?